app/bundles/InstallBundle/InstallFixtures/ORM/LeadFieldData.php

```diff
@@ -63,6 +63,7 @@ public function load(ObjectManager $manager)
             'facebook',
             'googleplus',
             'skype',
+            'linked',
```

Comment:
Should this be linkedin to match LoadLeadFieldData.php (and the two other places in this file)?